configs/projects/openvox-agent.rb

```diff
@@ -162,5 +162,11 @@
     proj.directory proj.bindir
   end
 
+  # make sure we can replace puppet-agent in place for the rename
+  if platform.is_rpm?
```

Comment:
I don't think we need this.  We'll want this on both rpm and deb packages.  In fact, I tested it with Debian 12 to make sure it was working.

Reply:
Updated.